h2o-r/tests/testdir_jira/runit_pubdev_2844.R

```diff
@@ -5,7 +5,9 @@ test.pubdev_2844 <- function() {
   
   df1 <- iris
   h2o.no_progress()
-  
+  # enable using of data.table for as.data.frame
+  options(h2o.fread=TRUE)
```

Comment:
does this also enable it for the rest of the test suite? or is it just this test?

Reply:
I think it probably does, will fix